suite-native/device-manager/src/components/WalletList.tsx

```diff
@@ -1,20 +1,59 @@
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
 
-import { selectDeviceInstances } from '@suite-common/wallet-core';
+import {
+    createDeviceInstanceThunk,
+    selectDevice,
+    selectDeviceInstances,
+    selectDeviceState,
+    selectIsPortfolioTrackerDevice,
+} from '@suite-common/wallet-core';
 import { VStack } from '@suite-native/atoms';
 import { TrezorDevice } from '@suite-common/suite-types';
 
 import { WalletItem } from './WalletItem';
+import { selectHasNonPassphraseDevice } from '@suite-native/device/src/selectors';
 
 type WalletListProps = {
     onSelectDevice: (device: TrezorDevice) => void;
 };
 
 export const WalletList = ({ onSelectDevice }: WalletListProps) => {
+    const dispatch = useDispatch();
     const devices = useSelector(selectDeviceInstances);
+    const selectedDevice = useSelector(selectDevice);
+    const hasNonPassphraseDevice = useSelector(selectHasNonPassphraseDevice);
+    const deviceState = useSelector(selectDeviceState);
+    const isPortfolioTrackerDevice = useSelector(selectIsPortfolioTrackerDevice);
+
+    const isSelectable = devices.length > 1 || !hasNonPassphraseDevice;
+
+    // we want to show placeholder in case there is only passphrase wallet without standard and not portfolio
+    const showPlaceholder = !hasNonPassphraseDevice && deviceState && !isPortfolioTrackerDevice;
+
+    // on tap of placeholder we actually create device with empty passphrase and select it
+    const handleOnPlaceholderPress = async () => {
+        if (selectedDevice) {
+            await dispatch(
+                createDeviceInstanceThunk({
+                    device: selectedDevice,
+                    useEmptyPassphrase: true,
+                }),
+            )
+                .unwrap()
+                .then(result => onSelectDevice(result.device));
```

Comment:
What if the `createDeviceInstanceThunk` fails?

Reply:
Not having standard wallet view only, but passphrase yes and wanting to get to standard one is already edge case, so imo we dont have to handle even more edge case scenario. Wdyt?

Reply:
Fair enough